test_generators/operations/main.py

```diff
@@ -1,9 +1,31 @@
+from eth2spec.test.block_processing import (
+    test_process_attestation,
+    test_process_attester_slashing,
+    test_process_block_header,
+    test_process_deposit,
+    test_process_proposer_slashing,
+    test_process_transfer,
+    test_process_voluntary_exit
+)
+
 from gen_base import gen_runner
 
-from deposits import mini_deposits_suite, full_deposits_suite
+from suite_creator import generate_from_tests, create_suite
 
 if __name__ == "__main__":
     gen_runner.run_generator("operations", [
-        mini_deposits_suite,
-        full_deposits_suite
+        create_suite('attestation',       'minimal', lambda: generate_from_tests(test_process_attestation)),
+        create_suite('attestation',       'mainnet', lambda: generate_from_tests(test_process_attestation)),
+        create_suite('attester_slashing', 'minimal', lambda: generate_from_tests(test_process_attester_slashing)),
+        create_suite('attester_slashing', 'mainnet', lambda: generate_from_tests(test_process_attester_slashing)),
+        create_suite('block_header',      'minimal', lambda: generate_from_tests(test_process_block_header)),
+        create_suite('block_header',      'mainnet', lambda: generate_from_tests(test_process_block_header)),
+        create_suite('deposits',          'minimal', lambda: generate_from_tests(test_process_deposit)),
```

Comment:
use singular `'deposit'` here  to conform to others

```suggestion
        create_suite('deposit',          'minimal', lambda: generate_from_tests(test_process_deposit)),
```